src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java

```diff
@@ -275,15 +285,24 @@ void updateSqlException() throws IOException, SQLException {
                 FileSearch fileSearch = mock(FileSearch.class);
                 fileSearchMock.when(() -> FileSearch.directory(ftpDirectory, true)).thenReturn(fileSearch);
 
-                Set<File> files = new HashSet<>(Set.of(mock(File.class), mock(File.class), mock(File.class)));
+                File file1 = Mockito.mock(File.class);
+                Mockito.when(file1.getName()).thenReturn("sample1.txt");
+                File file2 = Mockito.mock(File.class);
+                Mockito.when(file2.getName()).thenReturn("sample2.txt");
+                File file3 = Mockito.mock(File.class);
+                Mockito.when(file3.getName()).thenReturn("sample3.txt");
+                Set<File> files = new HashSet<>(Set.of(file1, file2, file3));
+
                 when(fileSearch.search()).thenReturn(files);
                 when(fileSearch.setFileNameRegexFilter(any())).thenReturn(fileSearch);
                 when(sftpService.upload(eq(sftpClass), any(File.class))).thenReturn(true);
 
                 SQLException sqlException = new SQLException("Forced sql exception");
                 for (File file : files) {
+                    String fileName = file.getName();
+                    when(file.getName()).thenReturn(fileName);
```

Comment:
This is not needed as it is already mocked

Reply:
Removed